### ISSUE TITLE
window fit + zooming doesn't change window size

### ISSUE THREAD

**Issue by zEdS15B3GCwq** [d2phap/ImageGlass#822]
### System information:
 - Windows OS version: Win 10 x64 1909
 - ImageGlass version: 7.6.4.30

### To Reproduce
1. enable "window fit" behaviour
2. open image, window fits image size as expected
3. zoom in

### Actual behavior:
Zooming in an image doesn't update window size to fit the new image size.

### Expected behavior:
Zooming in should update the window size, up until the window has room to grow. The current behaviour is not particularly useful nor is it reasonable - window fit shouldn't mean to only fit original size.

### Screenshots:
I think the description should be enough.

**Comment by d2phap** [comment]
It's design to work like this. Changed label

**Comment by zEdS15B3GCwq** [comment]
OK, thanks for the reply. I understand if it's working as designed, however, at least in my opinion, "window fit" suggests that the window would fit the size of the image in general, and not just when the image was opened. The current way of operation seems counter-intuitive. Opening a smallish image and zooming in to see details, I would expect the window to grow with the size of the image, instead of just keeping the window size and showing a small patch of the enlarged image.

Just to clear up any possible misunderstanding, I mean "window fit" and not "scale to fit", where the image scales to fit the window size.

**Comment by SylwesterZarebski** [comment]
I confirm. Zooming with `Window fit` option set should re-fit window to new image size.

**Comment by lyaaz** [comment]
yeah, hope to add this feature too. Option to fit window to image size  while zoom image or at least a hotkey to do this manually.

**Comment by mmuffins** [comment]
I agree, auto-fitting the window to the image size makes prevents having to constantly manually adjust the window size when zooming in and out a lot. For reference, see the following video:


https://user-images.githubusercontent.com/16214627/119728974-654d9880-be74-11eb-9fdf-36e15700ef08.mp4



The application used in the video is [jpegview](https://sourceforge.net/projects/jpegview/).

**Comment by SylwesterZarebski** [comment]
Is there any chance to code this in some foreseeable future? This is serious showstopper in my typical usage.

**Comment by SylwesterZarebski** [comment]
I've coded PR to fix this issue. Please review and possibly merge, thanks :-).

**Comment by SylwesterZarebski** [comment]
PS. I've also made PR #1241 to make `Window fit` more useful.

**Comment by SylwesterZarebski** [comment]
Thanks a lot :-).

**Comment by d2phap** [comment]
Shipped this to the latest build of IG Moon: https://imageglass.org/moon
